app/assets/javascripts/type_station/editables/text_html.js.coffee

```diff
@@ -23,6 +24,7 @@ class window.TS.EditableHtml
   constructor: (@elements) ->
     @editor = new MediumEditor @elements,
       buttons: ['bold', 'italic', 'underline', 'anchor', 'header1', 'header2', 'unorderedlist', 'orderedlist', 'justifyLeft', 'justifyFull', 'justifyCenter', 'justifyRight']
+      buttonLabels: 'fontawesome'
```

Comment:
Same again - "Should use Ionic Framework icons so that the user can use whatever version of fontawesome they like"